common/src/main/java/io/homeassistant/companion/android/common/data/prefs/WearPrefsRepositoryImpl.kt

```diff
@@ -51,6 +53,26 @@ class WearPrefsRepositoryImpl @Inject constructor(
 
                 localStorage.putInt(MIGRATION_PREF, MIGRATION_VERSION)
             }
+
+            if (currentVersion == 1) {
```

Comment:
This should be `< 2` to catch anyone that wasn't already migrated to 1 or future changes introduce another new version.

Reply:
Very good catch :+1: In my head, `currentVersion` was updated to `1` here if the previous block had been taken.. But that is of course not the case. Maybe the name of the variable mislead me :thinking:

Should we update the value of `currentVersion` at the end of each migration instead?

Reply:
Alternatively rename the variable to something else (e.g. `versionBeforeMigrations`).